Formula/zsh-fast-syntax-highlighting.rb

```diff
@@ -0,0 +1,29 @@
+class ZshFastSyntaxHighlighting < Formula
+  desc "Feature-rich syntax highlighting for Zsh"
+  homepage "https://github.com/zdharma-continuum/fast-syntax-highlighting"
+  url "https://github.com/zdharma-continuum/fast-syntax-highlighting.git",
+      tag:      "v1.55",
+      revision: "5351bd907ea39d9000b7bd60b5bb5b0b1d5c8046"
```

Comment:
Do we really need a Git clone?

Reply:
No, we don't. I was emulating the `zsh-syntax-highlighting` formula which did the same. We can use the usual Github tarball.